WcaOnRails/app/webpacker/components/Delegates/index.jsx

```diff
@@ -94,12 +111,11 @@ export default function Delegates() {
                     text: region.name,
                     value: region.id,
                   }))}
-                  value={activeRegion.id}
-                  onChange={(__, { value }) => setActiveRegion(
-                    delegateRegions.find((region) => region.id === value),
-                  )}
+                  value={hash}
+                  onChange={(__, { value }) => setHash(value)}
                 />
               </Grid.Row>
+              <Grid.Row><SelectedComponent /></Grid.Row>
```

Comment:
What does this do?

Reply:
It renders a component responsible for setting the default value. I followed what Daniel did in #8529

Reply:
The case in that PR is different. There, the component varies depending on the hash. But here, the component to be rendered is always the same - `DelegatesOfRegion`, which is already rendered. So, I guess this line is not needed. If you are also convinced, can you please remove this line and the initialization of `SelectedComponent`?